packages/kit/src/views/Swap/hooks/useSwapQuote.ts

```
@@ -466,6 +453,64 @@ export function useSwapQuote() {
+          setToTokenAmount({ value: toAmount, isInput: true });
+        } else {
+          setFromTokenAmount({ value: amount, isInput: true });
+        }
+      }
+    },
+    [
+      quoteAction,
+      setSwapSelectFromToken,
+      setSwapSelectToToken,
+      setFromTokenAmount,
+    ],
```

ESLint

[error] React Hook useCallback has missing dependencies: 'setToTokenAmount', 'swapTypeSwitchAction', and 'syncNetworksSort'. Either include them or remove the dependency array.

(react-hooks/exhaustive-deps)